src/containers/PipelineResources/PipelineResources.js

```diff
@@ -184,6 +198,22 @@ export /* istanbul ignore next */ class PipelineResources extends Component {
         />
       );
     }
+    const pipelineResourceActions = this.pipelineResourceActions();
+
+    let toolbarButtons = [
+      {
+        onClick: () => this.toggleModal(true),
+        text: intl.formatMessage({
+          id: 'dashboard.actions.createButton',
+          defaultMessage: 'Create'
+        }),
+        icon: Add
+      }
+    ];
+
+    if (this.state.isReadOnlyResult === 'true') {
+      toolbarButtons = [];
+    }
```

Comment:
we can flip this around

```suggestion
    const toolbarButtons = [];

    if (!isReadOnly) {
      toolbarButtons.push(
        {
          onClick: () => this.toggleModal(true),
          text: intl.formatMessage({
            id: 'dashboard.actions.createButton',
            defaultMessage: 'Create'
          }),
          icon: Add
        }
      );
    }
```